src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpUpgradeProject.cs

```diff
@@ -37,6 +37,7 @@ public void CPSProject_GeneralPropertyGroupUpdated()
 
             VisualStudio.SolutionExplorer.CreateSolution(SolutionName);
             VisualStudio.SolutionExplorer.AddProject(project, WellKnownProjectTemplates.CSharpNetStandardClassLibrary, LanguageNames.CSharp);
+            VisualStudio.SolutionExplorer.RestoreNuGetPackages(project);
```

Comment:
Any reason this should just be inlined into AddProject if you're adding a CSharpNetStandardClassLibrary?

Reply:
It might be possible to do that. If it wasn't such a slow operation we could apply it to all project additions.